plugins/ui/specs.nix

```diff
@@ -0,0 +1,139 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+with lib;
+helpers.neovim-plugin.mkNeovimPlugin config {
+  name = "specs";
+  originalName = "specs.nvim";
+  defaultPackage = pkgs.vimPlugins.specs-nvim;
+
+  maintainers = [ maintainers.GaetanLepage ];
+
+  # TODO: introduced 2024-06-10, remove on 2024-08-10
+  optionsRenamedToSettings = [
+    "show_jumps"
+    "min_jump"
+  ];
+  imports =
+    let
+      basePluginPath = [
+        "plugins"
+        "specs"
+      ];
+      settingsPath = basePluginPath ++ [ "settings" ];
+      renameToPopup =
+        old: new:
+        mkRenamedOptionModule (basePluginPath ++ [ old ]) (
+          settingsPath
+          ++ [
+            "popup"
+            new
+          ]
+        );
+    in
+    [
+      (renameToPopup "delay" "delay_ms")
+      (renameToPopup "increment" "inc_ms")
+      (renameToPopup "blend" "blend")
+      (renameToPopup "width" "width")
+      (mkRemovedOptionModule (
+        basePluginPath ++ [ "color" ]
+      ) "Please, use `settings.popup.winhl` directly.")
+      (mkRemovedOptionModule (
+        basePluginPath ++ [ "fader" ]
+      ) "Please, use `settings.popup.fader` directly.")
+      (mkRemovedOptionModule (
+        basePluginPath ++ [ "resizer" ]
+      ) "Please, use `settings.popup.resizer` directly.")
+      (mkRemovedOptionModule (
+        basePluginPath ++ [ "ignored_filetypes" ]
+      ) "Please, use `settings.ignore_filetypes` instead.")
+      (mkRemovedOptionModule (
+        basePluginPath ++ [ "ignored_buffertypes" ]
+      ) "Please, use `settings.ignore_buftypes` instead.")
```

Comment:
Are these all incompatible with the old options? _At first glance_ some of them look like they could be rename warnings instead of removal errors.

Reply:
No, they were all declared in a custom way before. Switching to a direct (RFC-42) style is breaking backward compatibility.